client/lib/flags/index.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import GridiconGlobe from 'gridicons/dist/globe';
+
+/**
+ * Given a country code, return a flag SVG file path
+ *
+ * @param   {String} countryCode  A two-letter ISO_3166-1_country code
+ * @returns {String}              Flag SVG file path
+ */
+export function flagUrl( countryCode ) {
+	try {
+		return require( `flag-icon-css/flags/4x3/${ countryCode }.svg` );
+	} catch ( e ) {
+		// As a fallback, return a 'globe' SVG.
+		// Unfortunately, we're not shipping SVGs with the `gridicons` npm --
+		// instead, they're inlined inside React components. The following code
+		// is thus needed to produce the SVG from the component.
+		// TODO: Ship SVGs with the `gridicons` npm, load the globe SVG directly.
+		const gridicon = <GridiconGlobe size={ 24 } />;
```

Comment:
Seems like we're providing a square icon as a fallback to a 4x3 icon. That's not ideal 😕 

Could we provide a 4x3 fallback? It could still be a globe, but with a 32x24 viewBox.

Reply:
Would simple empty box SVG work as well as a fallback?

```svg
<?xml version="1.0" encoding="UTF-8" ?>
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="32" height="24"></svg>
```

Reply:
Hmm, good point.

TIL about `viewBox`. Looks like there's no Gridicons prop to control that, and `size` only supports square proportions (fed into both the SVG's `width` and `height`, respectively) :slightly_frowning_face: 

So should I inline the SVG after all (with a manually adjusted `viewBox`)? :slightly_frowning_face:

Reply:
Oh, didn't see your comment when writing mine @simison 

We kinda want some kind of image here as a fallback I think -- if nothing else, then to retain current functionality of e.g. country selector dropdowns...

Reply:
> So should I inline the SVG after all (with a manually adjusted viewBox)? 

Sounds like a sensible solution for a fallback, considering all the good reasons to do it.